src/main/java/server/request/Request.java

```diff
@@ -0,0 +1,4 @@
+package server.request;
+
+public record Request(String method, String path, String protocol, String body) {
```

Comment:
Why do you need a `protocol` in Request? Is it going to be different?

Reply:
I didn't think I should hardcode the protocol version in case the server was responding to a client using a different version -- when I checked I saw that HTTP/2.0 was widely in use now. But should I make this HTTP/1.1 by default?

Reply:
Let's keep this simple and use HTTP/1.1 by default.

Reply:
Ok!